components/brave_rewards/resources/shared/components/newtab/select_country_card.style.ts

```diff
@@ -4,21 +4,17 @@
 
 import styled from 'styled-components'
 
-import * as mixins from '../../lib/css_mixins'
+import * as leo from '@brave/leo/tokens/css/variables'
```

Comment:
completely optional & up to you:

I find the Leo stuff normally gets automatically imported if you type `color<Ctrl+.>` and then you can use the slightly shorter `color.white` instead of `leo.color.white`.

Just a personal preference though, not really important :laughing: